src/components/DeleteCardForm.js

```diff
@@ -0,0 +1,28 @@
+import Popup from "./Popup";
+
+export default class DeleteCardForm extends Popup {
+  constructor(popupModalSelector, handleFormSubmit) {
+    super({ popupModalSelector });
+    this._popupModalSelector = document.querySelector(popupModalSelector);
```

Comment:
**No** need to find the popup again in the inheritors of class `Popup` because the parent has already found it in its constructor and it’s `this._popupModalSelector`. It’s available in any inheritor.